guides/doc-Content_Management_Guide/topics/QuickStart.adoc

```diff
@@ -92,3 +92,104 @@ Default_Organization_Centos_Stream_BaseOS_x86_64_os               BaseOS x86_64
+You can now register hosts to this content view and the hosts will have access to all the repositories at the locked version in the content view.
+
+This enables you to independently update repositories on the server while your hosts are still consuming the locked versions of the repositories in the published content views.
+
+If you do not use custom content views and lifecycle environments, all your hosts will always be pointed to the Default Content View and Library environment which is a snapshot of all the content at their latest version on the server.
+
+Please refer to the following chapters for more information on content management using content views and application life cycles.
+
+* xref:Managing_Content_Views[]
+* xref:Creating_an_Application_Life_Cycle[]
+
+==== Creating a Content View
+
+Use the following procedure to create a Content view.
+
+. In the {Project} web UI, navigate to *Content* > *Content Views* and click *Create New View*.
+. In the *Name* field, enter *Centos content view*.
+. In the *Description* field, enter a description for the content view.
+. Click *Save*.
+
+==== Adding Repositories
+
+Use the following procedure to add the upstream Centos repositories to your Content view.
+
+. In the {Project} web UI, navigate to *Content* > *Content Views* > Select *CentOS content view* > *Yum Content* Dropdown> *Repositories*.
+. Click on *Add* section on the Repository selection page.
+. Locate the *AppStream x86_64 os* and *BaseOS x86_64 os* repositories.
+. Select both repositories
+. Click *Add Repositories*
+
+==== Publishing the Content View
+
+Use the following procedure to publish the Content view.
+
+. In the {Project} web UI, navigate to *Content* > *Content Views* > *Select the desired Content View*
+. Click on *Publish New Version* button on the top right of page.
+. Click *Save*
+
+This will publish Version 1 of the Content View and promote it to the Library Environment.
+
+==== Creating an Activation Key to consume the Content View
+
+Use the following procedure to create an activation key that you can use to subscribe your hosts to content managed by {Project}.
+
+. In the {Project} web UI, navigate to *Content* > *Activation keys* and click *Create Activation Key*.
+. In the *Name* field, enter *CentOS* or the name of your new activation key.
+. In the *Description* field, enter a description for the activation key.
+. From the *Environment* list, select the *Library*.
+. From the *Content View* list, select *Centos content view*.
+. Click *Save*.
+
+. Click on the *Subscriptions* tab on the Activation Key's details page.
+. Click on the *Add* tab.
+. Select your new *CentOS Stream* product in the table.
+. Click *Add Selected* to save.
+
+==== Registering a CentOS Stream host to consume content from published Content View
+
+. You can follow steps in previous procedure Registering a CentOS Stream host to {Project} and use your new activation to register a host to the new Content View.
```

Comment:
I think that you can delete this bit, and add a paragraph after the last step of the previous procedure and say. 

`Use your activation key to register a host to the Content View. For more information on registering a host, see {ManagingHostsDocUR}registering-a-host-to-project-using-the-global-registration-template_managing-hosts`